technical/consensus/consensus.py

```diff
@@ -121,15 +121,15 @@ def evaluate_rsi(self, period=14, prefix="rsi", impact_buy=1, impact_sell=1):
 
         dataframe.loc[((dataframe[name] > 70)), f"sell_{name}"] = 1 * impact_sell
 
-    def evaluate_stoch(self, prefix="stoch", impact_buy=1, impact_sell=1):
+    def evaluate_stoch(self, period=14, prefix="stoch", impact_buy=1, impact_sell=1):
         """
-        evaluates a s
+        evaluates the stochastic fast
         :param dataframe:
         :param period:
         :param prefix:
         :return:
         """
-        name = f"{prefix}"
+        name = f"{prefix}_{period}"
```

Comment:
what's the point in passing in (and naming) a column with period=14 - if we're not using `period` to calculate anything?
is that a miss (wrongly copy/pasted into this method) - or something to align it with all other methods, so they all have a similar header (calling profile)?

Reply:
Good catch, thanks ! I've fixed that